RFC/RFC-14-p2p-adapter.md

```diff
@@ -416,3 +437,6 @@ Notice the peer relaying the message is not expected to remove itself either as
 - Why not let each peer calculate the best routing table, by broadcasting each peer status to the network?
   - This solution will consume a lot of resources from each client, since the table needs to be updated (or verified) every time there is a change.
   - This solution is somehow more difficult to debug, since routing tables have no source of truth, and different peers may build them differently. 
+- Can we replace the messaging service by providing alternatives routes for broadcasting a message?
+  - This won't solve the cluster partitions problem.
+  - By looking at the metrics of the messaging service usage, is possible to conclude the service is not solving a frequent enough problem, and then it can be removed or replaced.
```

Comment:
```suggestion
  - By looking at the metrics of the messaging service usage, it is possible to conclude that the service is not solving a frequent enough problem, and then it can be removed or replaced.
```